pkg/git/github/github_app.go

```diff
@@ -34,7 +34,8 @@ var NewGithubClientByApp func(appId int64, privateKeyPem []byte, repoUrl string)
 var NewGithubClientForSimpleBuildByApp func(appId int64, privateKeyPem []byte) (*GithubClient, error) = newGithubClientForSimpleBuildByApp
 
 var IsAppInstalledIntoRepository func(ghclient *GithubClient, repoUrl string) (bool, error) = isAppInstalledIntoRepository
-var GetAppInstallations func(githubAppIdStr string, appPrivateKeyPem []byte) ([]ApplicationInstallation, string, error) = getAppInstallations
+var GetAllAppInstallations func(githubAppIdStr string, appPrivateKeyPem []byte) ([]ApplicationInstallation, string, error) = getAppInstallations
+var GetAppInstallationsForRepository func(githubAppIdStr string, appPrivateKeyPem []byte, owner string, repo string) (*ApplicationInstallation, string, error) = getAppInstallationsForRepository
```

Comment:
For public functions the preferred way is to pass `repoUrl` and retrieve owner/repo inside. That would allow generic interface with other git providers.